rllib/core/rl_module/tf/tf_rl_module.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any, Mapping
+
+from ray.rllib.core.rl_module import RLModule
+from ray.rllib.core.rl_module.marl_module import MultiAgentRLModule
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_tf
+
+
+_, tf, _ = try_import_tf()
+
+
+class TFRLModule(RLModule, tf.keras.Model):
```

Comment:
Can we rename to "TfRLModule"? It's slightly more readable.
Otherwise, one might confuse it with the "TRFL" library from deepmind.